exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -270,9 +311,15 @@ func (prwe *prwExporter) export(ctx context.Context, requests []*prompb.WriteReq
 	return errs
 }
 
-func (prwe *prwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequest) error {
+func (prwe *prwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequest, writeReqv2 *writev2.Request, v2 bool) error {
```

Comment:
maybe just change this function to accept `data []byte`, rather than two different args?

Reply:
implemented